app/assets/javascripts/tags.coffee

```diff
@@ -0,0 +1,92 @@
+$(document).ready ->
```

Comment:
Since the process for getting tags for the show page and search page are different, I think adding a couple high level overview comments could help guide future developers to the right places.